todo_app/app.py

```diff
@@ -8,4 +13,11 @@
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    return render_template("index.html", items=get_items())
+
+
+@app.route('/add', methods=['POST'])
+def add():
+    if request.method == 'POST':
```

Comment:
You don't need this if as you already specify POST in the `@app.route` line above

Reply:
This has been removed and pushed in.